### PR TITLE
feat: added tests for GetUserByID function

### PR DESCRIPTION
**Pull request by vdshakh**

### CODE REVIEW

File: app/transport/rest/user_test.go [hunk 1]
```diff
@@ -202,3 +205,91 @@ func TestUserHandler_GetAllUsers(t *testing.T) {
...
+		usecaseBuilder        func(ctrl *gomock.Controller) UserUsecase
+		expectedStatusCode    int
+		expectedResponsetBody string
+	}{
+		"success": {
+			id: "91e3dcf7-34a6-4646-bd37-383cc949da93",
+			usecaseBuilder: func(ctrl *gomock.Controller) UserUsecase {
+				mock := NewMockUserUsecase(ctrl)
+
+				mock.EXPECT().GetByID("91e3dcf7-34a6-4646-bd37-383cc949da93").Return(&entities.User{
```
Comment (Brigant) [inline app/transport/rest/user_test.go:226]: Entity `user` has the field `password` and I think that the mock func should return that field too. The reason of the test is to ensure that the handle will not return the password even if the mock func will return it

File: app/transport/rest/user_test.go [hunk 2]
```diff
@@ -202,3 +205,91 @@ func TestUserHandler_GetAllUsers(t *testing.T) {
...
+			expectedStatusCode:    http.StatusNotFound,
+			expectedResponsetBody: `{"message": "Not found", "details": "no user found in DB"}`,
+		},
+		"internal_error": {
+			id: "10000000-0000-0000-0000-000000000000",
+			usecaseBuilder: func(ctrl *gomock.Controller) UserUsecase {
+				mock := NewMockUserUsecase(ctrl)
+
+				mock.EXPECT().GetByID("10000000-0000-0000-0000-000000000000").Return(nil,
+					globals.ErrDuplicateEmail).Times(1)
```
Comment (Brigant) [inline app/transport/rest/user_test.go:265]: I'm confused with `globals.ErrDuplicateEmail`. I thought the error is possible only while creating user. May be here will be better to create `errors.New()`

File: app/transport/rest/user_test.go (hunk 1 above)
Comment (vdshakh) [reply app/transport/rest/user_test.go:226]: Nice catch. Fixed it

File: app/transport/rest/user_test.go (hunk 2 above)
Comment (vdshakh) [reply app/transport/rest/user_test.go]: Fixed

**Review by Brigant — APPROVED** [review]

**Review by Rusli4k — APPROVED** [review]

**Review by Anton-Hudz — APPROVED** [review]

File: lgr/logger.go [hunk 3]
```diff
@@ -7,6 +7,11 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+const (
```
Comment (vpakhuchyi) [inline lgr/logger.go:10]: Could you please leave a comment to this constants block?

**Review by vpakhuchyi — APPROVED** [review]

**Review by Anton-Hudz — APPROVED** [review]

**Review by Brigant — APPROVED** [review]

**Review by YuriiYakovenko — APPROVED** [review]

**Pull request merged**